app/controllers/signatures_controller.rb

```diff
@@ -49,4 +54,12 @@ def correct_token(decoded_token)
   def signature_params
     params.require(:data).require(:signatures)
   end
+
+  def token_issued_at (decoded_token)
+    decoded_token['exp'] - ENV['IPAD_JWT_LENGTH'].to_i * 60
```

Comment:
🐘 there's an `'iat`' (issued at) claim we could set when the token is created which would make this nicer.

Reply:
cool - trying to reuse data to keep qr as small as possible